dev/tests/integration/testsuite/Magento/Framework/MessageQueue/UseCase/WaitAndNotWaitMessagesTest.php

```diff
@@ -66,14 +66,21 @@ protected function setUp(): void
      */
     public function testWaitForMessages()
     {
-        $this->assertArrayHasKey('queue', $this->config);
-        $this->assertArrayHasKey('consumers_wait_for_messages', $this->config['queue']);
-        $this->assertEquals(1, $this->config['queue']['consumers_wait_for_messages']);
+        $this->publisherConsumerController->stopConsumers();
+
+        $config = $this->config;
+        $config['queue']['consumers_wait_for_messages'] = 1;
```

Comment:
@le0n4ik Looks like we need to roll back to an original config value otherwise we broke the isolation

Reply:
@mslabko actually it's rolling back in 
\Magento\Framework\MessageQueue\UseCase\WaitAndNotWaitMessagesTest::tearDown

Config will be written from $this->config variable which was stored in setUp() function

Reply:
> @le0n4ik Looks like we need to roll back to an original config value otherwise we broke the isolation

@mslabko actually it's rolling back in
\Magento\Framework\MessageQueue\UseCase\WaitAndNotWaitMessagesTest::tearDown

Config will be written from $this->config variable which was stored in setUp() function